releases/values/teleport-ent/Install.md

```diff
@@ -0,0 +1,51 @@
+INSTALL checklist
```

Comment:
Lets standardize around the `README.md` which is automatically displayed when using GitHub.

Create an `## Install Checklist` chapter heading with the following content....